pages/treeherder.py

```diff
@@ -380,6 +378,15 @@ def click(self):
                 self.root.click()
                 self.wait.until(lambda _: self.page.job_details.job_result_status)
 
+    class InfoPanel(Region):
+
+        _info_panel_content_locator = (By.ID, 'info-panel-content')
+
+        @property
+        def info_panel_is_open(self):
+            el = self.find_element(*self._info_panel_content_locator)
+            return el.is_displayed()
```

Comment:
Unless we need to use the element in multiple places, or we're considering line lengths, I think it's better to just return the value directly. Note that the following assumes making the existing locator the root of the element as mentioned above.

```python
return self.root.is_displayed()
```